drupal/sites/default/boinc/modules/boincwork/includes/projectprefs.xsd

```diff
@@ -134,10 +134,10 @@
 
     <!-- grouping types -->
 
-    <xs:element name="apps">
+    <xs:element name="app_overrides">
         <xs:complexType>
             <xs:sequence>
-                <xs:element name="title" type="xs:string" minOccurs="1" maxOccurs="1" />
+                <xs:element name="title" type="xs:string" minOccurs="0" maxOccurs="1" />
```

Comment:
It sounds like there's no reason to keep it around. It will live on in our commit memory...